src/config/default-config.js

```diff
@@ -63,6 +63,7 @@ const config = {
   port: parseInt(process.env.PORT, 10) || 3000,
   clusterEnv: process.env.CLUSTER_ENV || 'development',
   sandboxId: process.env.SANDBOX_ID || 'default',
+  rdsClusterId: process.env.RDS_CLUSTER_ID || 'default',
```

Comment:
`RDS_CLUSTER_ID` is not the clusterId, it's actually the sandboxId but passed through the flag of stage an rds or not. I would suggest either passing the flag for stage an rds or not (so a true or false) or rename this to `RDS_SANDBOX_ID` or something like that.

Reply:
changed